src/frame/sdk/GroupApi.ts

```diff
@@ -84,18 +85,46 @@ export function get(groupID: string) {
     return GroupApiEndpoints.callGroupGetApi(groupID).map(formatDetailedGroupResponse);
 }
 
+function createErrorForInvalidUserList(userKeys: UserKeyListResponseType, userGrants: string[]) {
+    const existingUserIDs = userKeys.result.map(({id}) => id);
+    const missingUsersString = userGrants.filter((userID) => existingUserIDs.indexOf(userID) === -1);
+    return missingUsersString;
+}
+
 /**
  * Create a new group with the provided ID and options
  * @param {string}  groupID Client provided optional group ID
  * @param {string}  groupName Client provide optonal group name
  * @param {boolean} addAsMember Whether the group creator shoudl be added as a member upon create
  */
-export function create(groupID: string, groupName: string, addAsMember: boolean) {
-    return GroupOperations.groupCreate(ApiState.userPublicKey(), ApiState.signingKeys(), addAsMember)
-        .flatMap(({encryptedGroupKey, groupPublicKey, transformKey}) =>
-            GroupApiEndpoints.callGroupCreateApi(groupID, groupPublicKey, encryptedGroupKey, groupName, transformKey)
-        )
-        .map((createdGroup) => formatDetailedGroupResponse(createdGroup) as GroupDetailResponse);
+export function create(groupID: string, groupName: string, addAsMember: boolean, userList?: string[]): Future<SDKError, GroupDetailResponse> {
+    const creator = {id: ApiState.user().id, masterPublicKey: publicKeyToBase64(ApiState.userPublicKey())};
+    const creatorMemberKeys = addAsMember ? [creator] : [];
```

Comment:
```suggestion
    const creatorMemberKey = addAsMember ? [creator] : [];
```
Since it's only a single key